playbooks/openstack/openshift-cluster/provision.yml

```diff
@@ -14,7 +14,7 @@
 # and configure their DNS if they have to.
 
 - name: Prepare the Nodes in the cluster for installation
-  hosts: cluster_hosts
```

Comment:
this name change needs to be covered for the docs and the static_inventory and dynamic inventory please